src/backend/distributed/sql/citus--11.2-1--11.2-2.sql

```diff
@@ -0,0 +1,2 @@
+-- citus--11.2-1--11.2-2
+#include "udfs/worker_adjust_identity_column_seq_ranges/11.3-1.sql"
```

Comment:
I think the version portion of the file you edit and the file that is being included should match.

We can make an exception here as this is a backport. At least we can add some comments here explaining why the versions mismatch.